io.github.bgrabitmap.LazPaint.yml

```
@@ -0,0 +1,60 @@
+app-id: io.github.bgrabitmap.LazPaint
+runtime: org.kde.Platform
+runtime-version: '5.15-24.08'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.freepascal
+command: lazpaint
+build-options:
+    no-debuginfo: true
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+modules:
+  - name: qt5pas
+    buildsystem: qmake
+    config-opts:
+      - -after
+      - target.path=/app/lib
+    sources:
+      - type: shell
+        commands:
+          - cp -r /usr/lib/sdk/freepascal/share/lazarus/lcl/interfaces/qt5/cbindings/. .
+  - name: LazPaint
+    sources:
+      - type: git
+        url: https://github.com/bgrabitmap/bgrabitmap.git
+        commit: f9748a37f5382b18d1cd197293d1032a522e8561
+        dest: 'bgrabitmap'
+      - type: git
+        url: https://github.com/bgrabitmap/bgracontrols.git
+        commit: f5efbf8b79d0725d33b88c5bd41857d9139428bd
+        dest: 'bgracontrols'
+      - type: git
+        url: https://github.com/bgrabitmap/lazpaint.git
+        commit: 501d44c511fb49ca166dc0ec101305dddbf3aee6
+      - type: file
+        path: io.github.bgrabitmap.LazPaint.metainfo.xml
+      - type: file
+        path: io.github.bgrabitmap.LazPaint.desktop
+      - type: patch
+        path: 0001-515-runtime-fix-for-Qt.patch
+    buildsystem: simple
+    build-commands:
+      - |
+        . /usr/lib/sdk/freepascal/enable.sh
+        lazbuild --build-mode=Release --ws=qt5 bgrabitmap/bgrabitmap/bgrabitmappack.lpk
+        lazbuild --build-mode=Release --ws=qt5 bgracontrols/bgracontrols.lpk
+        lazbuild --build-mode=Release --ws=qt5 lazpaintcontrols/lazpaintcontrols.lpk
+        lazbuild --build-mode=Release --ws=qt5 lazpaint/lazpaint.lpi
+      - install -Dm755 lazpaint/release/bin/lazpaint -t $FLATPAK_DEST/bin
+      - install -d $FLATPAK_DEST/share/lazpaint
+      - install -Dm644 ${FLATPAK_ID}.desktop -t $FLATPAK_DEST/share/applications
+      - install -Dm644 resources/icon/256x256.png $FLATPAK_DEST/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png
+      - install -Dm644 lazpaint/release/bin/i18n/*.po -t $FLATPAK_DEST/share/lazpaint/i18n
+      - cp -r resources/scripts $FLATPAK_DEST/share/lazpaint
+      - cp -r lazpaint/release/bin/models $FLATPAK_DEST/share/lazpaint
+      - install -Dm644 Install/snap/local/lazpaint.xml -t $FLATPAK_DEST/share/mime/packages
```

```suggestion
      - install -Dm644 Install/snap/local/lazpaint.xml $FLATPAK_DEST/share/mime/packages/$FLAPTAK_ID.xml
```

otherwise it's useless and never exported.